### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by m-sameer**
**Changelog:**
- Added "_Table of Contents_" index
- Formated all the features, with _purpose_ and _usage_ sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,74 @@
...
+
+### Decide
+#### Purpose:
+Helping people in dilemma with 45% chances of the bot replying `Yes` (to do something, buy a new phone, relase your North Korean missile, etc.), 45% chances of replying `No`(no to downloading Vim, sing in shower, die because of not doing homework, etc.), and 10% chances of replying `Maybe`(putting people in dilemma again).
+#### Usage:
+`/decide`
+
+### Roll dice
+#### Purpose:
+Helping people play Ludo (who can't buy a physical kit or don't have the intellect to download an app for it) by outputing a random number between 1 and 6.
```
Comment (Yureien) [inline README.md]: Change from "Ludo" to "Board Games" and remove the don't have intellect part.

File: README.md [hunk 2]
```diff
@@ -3,17 +3,74 @@
...
+Markdown some of the messages that you'll be needing later.
+#### Usage:
+`[Reply to someone] /quote`: Append the message you replied to the list of quotes.
+
+`/quote [index]`: Show the quote on the index you typed.
+
+`/quote all`: List all the quotes ever.
+
+### Regex
+To be added later.
```
Comment (Yureien) [inline README.md]: Usage is like this -
Someone sent `String 1: Hello this is a string.`
You reply to String 1, with text - `s/Hello/Bye`.
The bot will reply `String 2: Bye this is a string.`
It uses the Python regex format, and is intended to be used like `sed`.

Purpose is to replace text with the bot.

**Comment by Yureien** [comment]
@m-sameer I have suggested some changes; please implement them, and I will acept this PR.

**Comment by m-sameer** [comment]
Doing. :smile:

**Comment by Yureien** [comment]
Good, and thanks for the PR.

**Comment by m-sameer** [comment]
Did the changes.

**Comment by Yureien** [comment]
Accepted.

**Comment by m-sameer** [comment]
Thanks for the merge. :)

**Pull request merged**